### PR TITLE
Working node to physics world conversions

### PR DESCRIPTION
**Pull request by vlidholt**
Still has a few todo items, but working for most part

### CODE REVIEW

File: examples/game/test_physics.dart [hunk 1]
```diff
@@ -64,23 +65,31 @@ class TestBed extends NodeWithSize {
...
+    //   new ActionTween((a) => _obstacle.scale = a, 2.0, 1.0, 2.0, Curves.easeInOut)
+    // ]);
+    // _obstacle.actions.run(new ActionRepeatForever(seq));
+
+    // Animate group
+    // ActionSequence seq = new ActionSequence([
+    //   new ActionTween((a) => _group.position = a, new Point(-256.0, 0.0), new Point(256.0, 0.0), 1.0, Curves.easeInOut),
+    //   new ActionTween((a) => _group.position = a, new Point(256.0, 0.0), new Point(-256.0, 0.0), 1.0, Curves.easeInOut)
+    // ]);
+    // _group.actions.run(new ActionRepeatForever(seq));
```
Comment (abarth) [inline examples/game/test_physics.dart]: Please don't land commented out code...

File: examples/game/test_physics.dart [hunk 2]
```diff
@@ -93,6 +102,8 @@ class TestBed extends NodeWithSize {
     if (event.type == "pointerdown") {
       Point pos = convertPointToNodeSpace(event.boxPosition);
 
+      print("ADDING SHIPS");
```
Comment (abarth) [inline examples/game/test_physics.dart]: Did you mean to land this print?

File: examples/game/test_physics.dart [hunk 3]
```diff
@@ -103,11 +114,11 @@ class TestBed extends NodeWithSize {
...
-      _physicsNode.addChild(shipA);
-      shipA.physicsBody.applyLinearImpulse(
-        new Offset(randomSignedDouble() * 5000.0, randomSignedDouble() * 5000.0),
-        shipA.position
-      );
+      _world.addChild(shipA);
+      // shipA.physicsBody.applyLinearImpulse(
+      //   new Offset(randomSignedDouble() * 5000.0, randomSignedDouble() * 5000.0),
+      //   shipA.position
+      // );
```
Comment (abarth) [inline examples/game/test_physics.dart]: ...

File: skysprites/lib/src/node.dart [hunk 4]
```diff
@@ -142,34 +142,63 @@ class Node {
...
+
+  PhysicsWorld _physicsWorld(Node parent) {
+    if (parent is PhysicsWorld) {
+      return parent;
+    }
+    else if (parent is PhysicsGroup) {
+      return _physicsWorld(parent.parent);
+    }
+    else {
+      print("physics not found");
```
Comment (abarth) [inline skysprites/lib/src/node.dart]: We should have a better way of signaling errors...

Comment (vlidholt) [inline skysprites/lib/src/node.dart]: Oops that was a left out debug print, I'll replace it with an assert

**Comment by abarth** [comment]
LGTM

**Pull request merged**